### PR TITLE
Add missing user session attributes

### PR DESCRIPTION
**Pull request by rroupski-splunk**
[PR 462](https://github.com/ocsf/ocsf-schema/pull/462) removed the `session_uid` and `session_uuid` from the `user` object, thus we lost the attributes used for logging the session related data.

Add `session` attribute, which pairs with the `user` attribute, to `process` object, `authentication`, and `authorization` events. 

Update the session caption and description.

Signed-off-by: Roumen Roupski <rroupski@splunk.com>

### CODE REVIEW

File: events/audit/authentication.json [hunk 1]
```diff
@@ -41,8 +27,7 @@
     },
     "actor": {
       "description": "The actor that requested the authentication.",
-      "group": "context",
-      "profile": null
```
Comment (floydtree) [inline events/audit/authentication.json:45]: Let's keep `profile: null`. The actor object should be by default a part of this class. Doesn't make sense to use host profile for non-host based activities. (actor is a part of the host profile as of today)

**Review by floydtree — CHANGES_REQUESTED** [review]
Let's keep `profile: null`. The actor object should be by default a part of this class. Doesn't make sense to use host profile for non-host based activities. (actor is a part of the host profile as of today)

**Review by mr-splunk — COMMENTED** [review]
@rroupski-splunk aren't we also going to add 'session' to the 'Account Change' events for consistency?

image

UPDATE: Roumen and I resolved this one offline. There is no demonstrable use-case for adding session to 'Account Change'. If it ever needs to be added in the future, that is fine as it would be an addition and as such a non-breaking change to OCSF.

**Review by pagbabian-splunk — DISMISSED** [review]
The updated descriptions should impart the proper usage, when `user` is paired with `session`: e.g. when a process is running under the account of a `user` the process session is the user session.  Hence in Actor, where either `user` OR `process` is populated, `session` is only populated when `user` is populated, since `process` has a `user' and `session` pair within Process.

File: events/audit/authentication.json (hunk 1 above)
Comment (rroupski) [reply events/audit/authentication.json:45]: OK, we can keep it, just to confirm --  it means you have to have the actors for all cloud events.

**Review by mr-splunk — DISMISSED** [review]
This looks good to me, assuming that the `profile: null` conversation is worked out.

File: events/audit/authentication.json (hunk 1 above)
Comment (floydtree) [reply events/audit/authentication.json:45]: Yes, it's okay to have `actor` available in this regardless even if Host profile is not used.

**Review by floydtree — APPROVED** [review]

**Review by rroupski — APPROVED** [review]

**Review by mr-splunk — APPROVED** [review]

**Pull request merged**